onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/serialization/mappings/FareProductFieldMappingFactory.java

```diff
@@ -47,13 +47,13 @@ public FieldMapping createFieldMapping(EntitySchemaFactory schemaFactory,
     return new FareProductFieldMapping(entityType, csvFieldName, objFieldName, required);
   }
 
-  public static AgencyAndId fareProductId(String agencyId, String fareProductId, String riderCategoryId, String fareContainerId) {
-    String primaryKey = fareProductIdPrimaryKey(fareProductId, riderCategoryId, fareContainerId);
+  public static AgencyAndId fareProductId(String agencyId, String fareProductId, String riderCategoryId, String fareMediumId) {
+    String primaryKey = fareProductIdPrimaryKey(fareProductId, riderCategoryId, fareMediumId);
     return new AgencyAndId(agencyId, primaryKey);
   }
 
-  static String fareProductIdPrimaryKey(String fareProductId, String riderCategoryId, String fareContainerId) {
-    return String.format("id=%s|category=%s|container=%s", fareProductId, riderCategoryId, fareContainerId);
+  static String fareProductIdPrimaryKey(String fareProductId, String riderCategoryId, String fareMediumId) {
```

Comment:
I know I originally wrote this code but can we rename this method to `fareProductPrimaryKey`?

Shame on me...